pkg/resolution/resolver/git/resolver.go

```diff
@@ -247,9 +251,15 @@ var _ framework.TimedResolution = &Resolver{}
 // GetResolutionTimeout returns a time.Duration for the amount of time a
 // single git fetch may take. This can be configured with the
 // fetch-timeout field in the git-resolver-config configmap.
-func (r *Resolver) GetResolutionTimeout(ctx context.Context, defaultTimeout time.Duration) time.Duration {
-	conf := framework.GetResolverConfigFromContext(ctx)
-	if timeoutString, ok := conf[DefaultTimeoutKey]; ok {
+func (r *Resolver) GetResolutionTimeout(ctx context.Context, defaultTimeout time.Duration, params map[string]string) time.Duration {
```

Comment:
I see same logic and function GetResolutionTimeout  here [pkg/remoteresolution/resolver/git/resolver.go](https://github.com/tektoncd/pipeline/pull/8263/files?show-viewed-files=true&file-filters%5B%5D=#diff-ecb3402dfe5b1846a7e8733f7018828293186ce7040d997973def09fce3d5cb7R144-R159)  as well  can we reuse instead of keeping in 2 places ?